cmd/login.go

```diff
@@ -61,9 +67,10 @@ func Login() *cobra.Command {
 
 			authorizationURL := buildAuthorizationURL(handler.baseURL, handler.state, port)
 			fmt.Println("Authentication will continue in your default browser")
-			if err := open.Start(authorizationURL); err != nil {
-				fmt.Printf("Please open a browser and navigate to the following address:\n")
-				fmt.Println(authorizationURL)
+			if showUrl {
```

Comment:
simple if condition1 or condition2 could help. But did not work with my lack of knowledge of Go.